modules/table/src/test/java/org/apache/ignite/internal/table/distributed/SortedIndexLockerTest.java

```diff
@@ -61,7 +61,7 @@ void takingInsertLocksOnDestroyedIndexStorageYieldsNullLock() {
         );
         indexStorage.destroy();
 
-        SortedIndexLocker locker = new SortedIndexLocker(1, PARTITION_ID, new HeapLockManager(), indexStorage, row -> binaryTuple);
+        SortedIndexLocker locker = new SortedIndexLocker(1, PARTITION_ID, new HeapLockManager(), indexStorage, row -> binaryTuple, false);
```

Comment:
Do we test only non-unique index here?
Is this class necessary after you added PartitionReplicaListenerSortedIndexLockingTest?